docs/1.3/customization/event-dispatcher.md

```diff
@@ -69,6 +69,10 @@ Listeners may call any method on the event to get more information about the eve
 
 This library supports the following default events which you can register listeners for:
 
+### `League\CommonMark\Event\DocumentPreParsedEvent`
+
+This event is dispatched just before any processing is done. It can be used to pre-populate reference map of a document before any processing is performed.
+
```

Comment:
I've added the `docs/1.4` folder.  Would you mind rebasing this PR onto `master` and apply this change to `1.4` instead?

Reply:
Done!